distribution/docs/src/main/resources/_contents/_configuring/configuring-global-settings-contents.adoc

```diff
@@ -224,6 +224,30 @@ TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
 |org.apache.xerces.jaxp.DocumentBuilderFactoryImpl
 |Yes
 
+
+6+^h|Catalog Source Retry Interval
+
+|Initial Endpoint Contact Interval
+|org.codice.ddf.platform.util.http.initialRetryInterval
+|Integer
+|If a Catalog Source is unavailable, try to connect to it after the initial interval has elapsed.
+The interval doubles after each try, up to a given maximum interval.
+The interval is measured in seconds.
+|10
+|Yes
+
+|Maximum Endpoint Contact Interval
+|Maximum seconds between attempts to establish contact with unavailable Catalog Source.
+|Integer
+|Do not wait longer than the maximum interval to attempt to establish a connection with an
+unavailable Catalog Source. Large values result in more network traffic trying to contact
```

Comment:
This wording is confusing in my mind. It implies that a longer max retry interval will result in more total network traffic than a lower max retry interval.